File: ACE Hardware/ACE Hardware/ReportsForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace ACE_Hardware
{
    public partial class ReportsForm : Form
    {

        private ReportsSalesReportForm salesreport = new ReportsSalesReportForm();
        private ReportsProductSalesReportForm productsalesreport = new ReportsProductSalesReportForm();

        public ReportsForm()
        {
            InitializeComponent();
        }

        private void Button2_Click(object sender, EventArgs e)
        {
            Program.mainFormInstance.Show();
            this.Close();
        }

        private void ReportsForm_Load(object sender, EventArgs e)
        {
            salesreport.TopLevel = false;
            salesreport.Dock = DockStyle.Fill;
            productsalesreport.TopLevel = false;
            productsalesreport.Dock = DockStyle.Fill;
            Panel1.Controls.Add(salesreport);
            salesreport.Show();
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            Panel1.Controls.Clear();
            Panel1.Controls.Add(salesreport);
            salesreport.Show();
        }

        private void ReportsForm_FormClosed(object sender, FormClosedEventArgs e)
        {
            Program.mainFormInstance.Show();
        }

        private void Button3_Click(object sender, EventArgs e)
        {
            Panel1.Controls.Clear();
            Panel1.Controls.Add(productsalesreport);
            productsalesreport.Show();
        }
    }
}

File: POS/POS/ReportsForm.vb
Public Class ReportsForm
    Dim salesreport As New ReportsSalesReportForm
    Dim productsalesreport As New ReportsProductSalesReportForm
    Private Sub ReportsForm_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        salesreport.TopLevel = False
        salesreport.Dock = DockStyle.Fill
        productsalesreport.TopLevel = False
        productsalesreport.Dock = DockStyle.Fill
        Panel1.Controls.Add(salesreport)
        salesreport.Show()
    End Sub

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Panel1.Controls.Clear()
        Panel1.Controls.Add(salesreport)
        salesreport.Show()
    End Sub

    Private Sub ReportsForm_Closed(sender As Object, e As EventArgs) Handles Me.Closed
        MainForm.Show()
    End Sub

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        MainForm.Show()
        Me.Close()
    End Sub

    Private Sub Button3_Click(sender As Object, e As EventArgs) Handles Button3.Click
        Panel1.Controls.Clear()
        Panel1.Controls.Add(productsalesreport)
        productsalesreport.Show()
    End Sub
End Class